openrtb_ext/imp_marsmedia.go

```diff
@@ -0,0 +1,8 @@
+package openrtb_ext
+
+// ExtImpMarsmedia defines the contract for bidrequest.imp[i].ext.marsmedia
+type ExtImpMarsmedia struct {
+	Publisher	string	`json:"publisher"`
+	ZoneId		string	`json:"ZoneId"`
+	BidFloor	float64	`json:"bidfloor"`
```

Comment:
Parameters  `ZoneId` and `BidFloor` don't get used anywhere inside `adapters/marsmedia/`. Are we sure we need them?